starknet_py/net/models/transaction_payloads.py

```diff
@@ -0,0 +1,166 @@
+from abc import ABC
```

Comment:
I think `payloads` is misleading. These are transactions returned by starknet, not ones that can be sent.

Reply:
responses?

Reply:
I think that's better